docs/scenarios_authoring/BasicNodes.md

```diff
@@ -4,77 +4,75 @@ sidebar_position: 2
 &nbsp;
 ## Variable
 
-A Variable component is used to declare a new variable; in the simplest form a variable declaration looks like in the example  below. As the event was read from the Kafka topic, the `#input` variable holds its content and  its value is assigned to a newly declared `myFirstVariable` variable.
+A Variable component is used to declare a new variable. In its simplest form a variable declaration looks like the example  below. As the event has been read from a Kafka topic, the `#input` variable holds event contents. Event value is assigned to a newly declared `myFirstVariable` variable.
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_variable0.png   "Scenario with variable declaration")
 
 
-As you can see in the `variable` configuration form below, Nussknacker inferred the data type of the `#input` variable from the information already available to Nussknacker.
+As you can see in the `variable` configuration form below, Nussknacker has inferred the data type of the `#input` variable. Nussknacker can do this based on info stored in the previous components.
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_variable1.png "Variable declaration form")
 
 
-In the next example `#input` variable is used to create an expression returning a boolean value. If the input Kafka topic contains JSON objects and they contain `operation` field, the value of this field can be obtained in the following way:
+In the next example `#input` variable is used to create an expression returning a boolean value. If the input Kafka topic contains JSON objects and they contain an `operation`, the value can be obtained using the following pattern:
 
 
 `#input.operation`
 
-Note that internally Nussknacker converts JSON’s object into SpEL’s map.
-
-
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_variable2.png "Screenshot_tooltip")
 
+Note that internally Nussknacker converts the JSON object into a SpEL map.
 &nbsp;
 ## RecordVariable
 
-The specialized `record-variable` component can be used to declare a record variable (object in JSON)
+The specialized `record-variable` component can be used to declare a record variable (JSON object)
 
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_mapmariable0.png "record-variable form")
 
 
-The same can be achieved using a plain `Variable` component, just make sure to write a valid SpEL expression.
+The same outcome can be achieved using a plain `Variable` component. Just make sure to write a valid SpEL expression.
 
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_mapmariable1.png "record-variable declaration using a plan Variable component")
 
 &nbsp;
 ## Filter
 
-Filter passes records which satisfy the filtering condition to `true sink`.
+Filters let through records that satisfy a filtering condition.
```

Comment:
maybe "Filters let through only records ..."

Reply:
Isn't that tad close to a tautology? :D

Reply:
User might think that they let through also some other kind of records :) But on the other hand no one is gong to think that so it can stay